### ISSUE TITLE
Prepare for native test explorer

### ISSUE THREAD

**Issue by razzeee** [elm-tooling/elm-language-client-vscode#185]
Vscodes native test explorer is coming up in one of the next versions. Our migration path is likely going to be via https://marketplace.visualstudio.com/items?itemName=ms-vscode.test-adapter-converter

See: https://code.visualstudio.com/updates/v1_56#_testing

**Comment by l2dy** [comment]
Anything to be done in this project if we use the converter extension? Migration is only needed if we switch to the native API.

**Comment by razzeee** [comment]
I don't think we need to do anything for the converter, but I would like to migrate to the native api, as it's supposed to have more features/better integration. But I'm not sure if we will reap many benefits in Elm due to that. Some of those features might be connected to debugging for e.g.

**Comment by l2dy** [comment]
From https://marketplace.visualstudio.com/items?itemName=hbenl.vscode-test-explorer:

> In version 1.59, VS Code added an official API and UI for running tests, which provides all the functionality of this extension and more. Therefore this extension is now deprecated. [...]

The official API is finalized in version 1.59. See https://code.visualstudio.com/updates/v1_59#_testing-apis.

The migration would at least reduce the number of deprecated extensions installed ;)

> You can keep using this extension as before, but you now also have the option to use VS Code's native testing UI instead by setting `testExplorer.useNativeTesting` to `true` in your VS Code settings.

Should we encourage users to try this out?

**Comment by razzeee** [comment]
I think so, the experience should be better. I still think we should migrate, but somebody would need to invest the time.

**Comment by razzeee** [comment]
https://code.visualstudio.com/api/extension-guides/testing#migrating-from-the-test-explorer-ui

**Comment by kostisvilaetis** [comment]
Hello, are there any plans to remove the Test Explorer UI hard dependency now that the native test explorer is out?

**Comment by razzeee** [comment]
Not sure if we can
